style.css

```diff
@@ -1,290 +1,905 @@
+  margin: 0;
+  padding-top: 20%;
+}
+
+#aboutmetext {
+  width: 90vw;
+  padding-bottom: 5%;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  margin-bottom: 5%;
+}
+
+.list3 {
+  padding-top: 13%;
+}
+
+.getresumebutton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  background: #ff6b00;
+  width: 45vw;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  border: none;
+}
+
+.lang {
+  color: white;
+  width: 70vw;
+  align-content: center;
+  justify-content: center;
+  margin: 20vh auto 0 auto;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  column-gap: 10px;
+}
+
+.skilllist {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  flex-wrap: wrap;
+
+  /* height: 150vh; */
+}
+
+#first::before {
+  content: url('image/Rectangle\ 55.png');
+  margin-left: 2vw;
+}
+
+#first::after {
+  margin-left: 10vw;
+}
+
+#second::before {
+  content: url('image/Rectangle\ 56\ \(1\).png');
+  padding-left: 5vw;
+}
+
+#third::before {
+  content: url('image/Rectangle\ 57.png');
+  padding-left: 2vw;
+}
+
+.listitem {
+  width: 50%;
+  height: 60vh;
+  margin: 0 auto 5% auto;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+}
+
+#first {
+  margin-top: 15vh;
+}
+
+.list {
+  text-align: center;
+  line-height: 43px;
+  color: #fff;
+  padding-top: 5%;
+  padding-bottom: 10%;
+  font-weight: normal;
+  font-size: 18px;
+  padding-left: 15%;
+}
+
+#second {
+  margin-top: 7vh;
+}
+
+#third {
+  margin-top: 7vh;
+}
+
+.divider {
+  width: 90%;
+  margin: auto;
+  border: 0.5px thin #fff;
+  margin-top: 13%;
+}
+
+.contact-form {
+  padding-bottom: 10%;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-color: white;
+  margin-bottom: 17%;
+}
+
+.footer {
+  background-color: white;
+  padding-bottom: 2%;
+}
+
+.footer .item {
+  color: #505f79;
+}
+
+.footer .liner {
+  height: 4px;
+  width: 145px;
+  margin: 0 auto;
+  background: #0d151e;
+  border-radius: 10px;
+}
+
+button {
+  transition: all 1s ease;
+}
+
+button:enabled {
+  background: #f55800;
+}
+
+button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  cursor: pointer;
+  transform: scale(1.1);
+  color: #f55800;
+}
+
+button:active {
+  background: #e05304;
+}
+
+button:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+.contact-form p {
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 90vw;
+  margin: 20% auto 10% auto;
+  text-align: center;
+  flex: 1;
+}
+
+.mobile {
+  display: none;
+}
+
+.form_details {
+  display: flex;
+  flex-direction: column;
+}
+
+input {
+  width: 90%;
+  padding: 2%;
+  margin: 3%;
+  border: 0;
+  border-bottom: 2px solid #dbd8d7;
+}
+
+textarea {
```

Comment:
- [ ] [OPTIONAL] Good job adding padding on the placeholder, it would be great if you do the same on the `textarea` as it will make it more consistent with the design and result in a better user experience. 😎️